docs/refresh_symlinks.sh

```diff
@@ -4,3 +4,6 @@
 # and the docs won't build properly.
 export MSYS=winsymlinks:nativestrict
 ln -srf ../README.md readme.md
+ln -srf ../CONTRIBUTING.md contributing.md
+ln -srf ../material/cheat_sheets.md material/cheat_sheets.md
+ln -srf ../material/tutorials.md material/tutorials.md
```

Comment:
seems like links to the files themselves

Reply:
They link from `<repo_root>/material` to `<repo_root>/docs/material`